### ISSUE TITLE
apps with icase AppImage are not listed

### ISSUE THREAD

**Issue by wesskywalker** [prateekmedia/appimagepool#81]
Some apps don't use the extension `.AppImage`, but they would rather user `.appImage`, `.Appimage`, `.appimage` or any other upper/lowercase combination.
No idea if this is canonical but it's what is happening.
Examples with avidemux:

- https://www.appimagehub.com/p/1238389
- https://github.com/mean00/avidemux2/releases
- https://sourceforge.net/p/avidemux/bugs/111/

Other software like `appimagelauncherd` use case insensitive so read whatever.

**Comment by prateekmedia** [comment]
I don't think so, if the app is not available in github then we don't have a way to get it.

**Comment by wesskywalker** [comment]
Probably you are misunderstanding what I mean.
Download this file: `https://github.com/mean00/avidemux2/releases/download/2.8.0/avidemux_2.8.0.appImage` and place it inside `$HOME/Applications`. It will not be detected by `appimagepool`.

**Comment by wesskywalker** [comment]
Use this:
```
wget https://github.com/mean00/avidemux2/releases/download/2.8.0/avidemux_2.8.0.appImage -O $HOME/Applications/avidemux_2.8.0.appimage
```

**Comment by prateekmedia** [comment]
Make it executable using
`chmod +x whatever`

**Comment by prateekmedia** [comment]
I will try this appimage once I am on my PC

**Comment by wesskywalker** [comment]
I don't think it's related to exec permission.

**Comment by lfom** [comment]
This still happens in v5.1.0. I was looking for a simple app to integrate downloaded AppImage files with the DE and found this issue. Readme says AppImagePool can manage integration but there is no option to open/import the files, the user must first copy the file default download path that curiously is ~/Applications instead of ~/Download. Anyway it seems that it only detects files that end in .AppImage, ignoring anything else with the same extension but diffrence case. It is _not_ related to file permssions.

**Comment by prateekmedia** [comment]
You can Infact change directory of AppImages but not auto appimage detect is possible for now.

**Comment by lfom** [comment]
> You can Infact change directory of AppImages but no systemwide auto appimage detect is possible for now.

Thank you for your quick reply. The default directory is not a big deal in my opinio (I will open a new issue about adding a feature so the user can open AppImage files from anywhere, what I think will improve the user experience).

What I think is more important is to fix the auto-detection of files with any extension, what seems to be broken as of now, at least in the latest release version:
- .AppImage (works)
- .appimage (is _not_ detected)
- .appImage (is _not_ detected)
- .Appimage (is _not_ detected)

Cheers
